endpoints/api/__init__.py

```diff
@@ -64,7 +64,7 @@ def handle_error(self, error):
 api = ApiExceptionHandlingApi()
 api.init_app(api_bp)
 api.decorators = [
-    csrf_protect(),
+    csrf_protect(skip_paths=["/api/v1/user/initialize"]),
```

Comment:
Instead of hacking in the current decorator for one specific use case, if your api needs a different set of decorators than these ones, you should create a separate blueprint for your resource.

Reply:
Also, for this case, you can probably make the request on behalf of Quay, and use  `app.config["HTTPCLIENT"]` to go around the CRSF protection.

Reply:
Do you have example of HTTPCLIENT and how it would be used?